lua/null-ls/utils/init.lua

```diff
@@ -211,6 +211,20 @@ M.handle_function_opt = function(opt, ...)
     return vim.deepcopy(opt)
 end
 
+--- gets git root
+---@return string
+M.get_git_root = function()
+    local result = vim.system({
```

Comment:
prefer `vim.loop.spawn` here?

Reply:
also: traverse filesystem first for vcs directories (`.git`, `.svn`, `.bzr`, etc.) before spawning a new process.

Reply:
> prefer `vim.loop.spawn` here?

Seems spawn is async func, changed to `vim.fn.system`.

Reply:
traverse is added.